oxide.json

```
@@ -16101,10 +16147,15 @@
           }
         ]
       },
-      "LldpServiceConfig": {
+      "LldpLinkConfig": {
         "description": "A link layer discovery protocol (LLDP) service configuration.",
         "type": "object",
         "properties": {
+          "chassis_id": {
+            "nullable": true,
+            "description": "The LLDP chassis identifier TLV.",
```

Should we define TLV, like LLDP is above?

```suggestion
            "description": "The LLDP chassis identifier type-length-value (TLV).",
```

---

This comes from the Omicron API definition and is simply imported here.